higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -292,33 +302,71 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
     showHiddenInputs(selected_province.value, uniClone, uni, universities);
     showHiddenInputs(uni.options[uni.selectedIndex].value, campClone, camp, campuses);
     clearCampus();
+    showOther(selected_province.value, universities, uni_other);
+    showOther(uni.options[uni.selectedIndex].value, campuses, camp_other);
+
     prov.addEventListener('change', function() {
       showHiddenInputs(this.value, uniClone, uni, universities);
       showHiddenInputs(uni.value, campClone, camp, campuses);
       clearCampus();
     });
     uni.addEventListener('change', function() {
       showHiddenInputs(this.value, campClone, camp, campuses);
+      camp.parentElement.classList.remove('hidden');
+      showOther(this.value, universities, uni_other);
+    });
+    camp.addEventListener('change', function() {
+      showOther(this.value, campuses, camp_other);
```

Comment:
`showOther` compares the first argument to `other_university`, but here, `this.value` will be a campus, not a university, so it's confusing why we're passing it to `showOther`.